src/curated-corpus/components/ApprovedItemListCard/ApprovedItemListCard.tsx

```diff
@@ -148,42 +149,42 @@ export const ApprovedItemListCard: React.FC<ApprovedItemListCardProps> = (
       >
         {item.isSyndicated && (
           <ListItem>
-            <StyledListItemIcon>
+            <ListItemIcon sx={{ minWidth: '2rem' }}>
               <CheckCircleOutlineIcon fontSize="small" />
-            </StyledListItemIcon>
+            </ListItemIcon>
```

Comment:
Have had to move away from using a styled component back to `sx` props that apply CSS styles directly here and in a couple of other components. Added a comment explaining why so that folks who may think it's a good idea to refactor this in the future are forewarned.